source/PlayState.hx

```diff
@@ -20,6 +22,15 @@ class PlayState extends FlxState
 		player = new Player(50, 50);
 		add(player);
 		
+		//Coins are added to a group, coin group added to playstate
+		coins = new FlxGroup();
+		coins.add(Coin.createCoin(8, 8, true));
+		coins.add(Coin.createCoin(13, 8, true));
```

Comment:
You might consider changing the third parameter for the Coin classes's constructor to be something like a FlxColor. That way it might be a little more clear as to what the third argument is doing here without having to go look inside of the Player class itself